docs/source/getting-started/installation.rst

```diff
@@ -151,3 +151,19 @@ Finally, you can stop the container with:
 .. code-block:: console
 
   $ docker stop sky
+
+.. _shell-completion:
+
+Enabling shell completion
+-------------------------
+
+SkyPilot supports shell completion for Bash (Version 4.4 and up), Zsh and Fish.
+
+To enable shell completion after installing SkyPilot, you will need to modify your shell configuration.
+SkyPilot automates this process using the :code:`--install-shell-completion` option, which you should call using the appropriate shell name:
+
+.. code-block:: console
```

Comment:
We need to have a way to uninstall it. [Reference for the terraform autocomplete](https://www.terraform.io/cli/commands#shell-tab-completion)

Also, can we automatically detect the shell the command is running in by default when the `zsh/bash/fish` is not provided?